### PR TITLE
Add org.tryton.Tryton-68

### PR DESCRIPTION
**Pull request by cedk**
This is the same as org.tryton.Tryton-66 but for the series 6.8

### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by hfiguiere** [comment]
I would have expected that by now the .desktop, icon and appsteream would come directly from upstream.

File: org.tryton.Tryton-68.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+runtime: org.gnome.Platform
+runtime-version: '44'
+command: tryton
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=all
+  - --share=network
+  - --talk-name=org.freedesktop.Notifications
```
Comment (hfiguiere) [inline org.tryton.Tryton-68.yml:12]: if you use GNotification, you shouldn't need this dbus permission

**Comment by hfiguiere** [comment]
bot, build org.tryton.Tryton-68

**Comment by cedk** [comment]
> I would have expected that by now the .desktop, icon and appstream would come directly from upstream.

The desktop and icon files are different because the application name is not the same.
For appstream also the application name is different but also there is no point to manage it upstream.

**Comment by hfiguiere** [comment]
> > I would have expected that by now the .desktop, icon and appstream would come directly from upstream.
> 
> The desktop and icon files are different because the application name is not the same. 

There is `desktop-file-edit` to fix that. Short of actually making it the same everywhere (nothing is flathub specific, just that we mandate the use of the updated convetions)

> For appstream also the application name is different but also there is no point to manage it upstream.

Yes there is. It's not flatpak specific either.

File: org.tryton.Tryton-68.yml (hunk 1 above)
Comment (cedk) [reply org.tryton.Tryton-68.yml:12]: It is using https://docs.gtk.org/gio/method.Application.send_notification.html

**Comment by cedk** [comment]
> like other python app by installing the app from the repo. Or just getting said files from there.

Releases are published because the repository does not contain the generated files.
Also the repository is 100 times bigger than the release archive.

**Comment by hfiguiere** [comment]
bot, build org.tryton.Tryton-68

**Comment by barthalion** [comment]
If the package is considered official, please verify it at https://flathub.org/login when it becomes visible on the website in a few hours.

**Pull request closed without merging**